### PR TITLE
Add display and copy MFA key

### LINKED ISSUE #713: [Request] Add secret key for 2fa under barcode.
This way, it will be easier for those of us who can just copy paste the key into our password manager!

### PR DESCRIPTION
**Pull request by princemaple**
close #713 

image

image

### CODE REVIEW

File: apps/fz_http/lib/fz_http_web/live/mfa_live/register_steps_component.ex [hunk 1]
```diff
@@ -70,6 +70,10 @@ defmodule FzHttpWeb.MFA.RegisterStepsComponent do
 
       <div class="has-text-centered">
         <canvas data-qrdata={@uri} id="register-totp" phx-hook="RenderQR" />
+
+        <div class="mb-4">
+          <a href="#" id="copy-totp-uri" phx-hook="ClipboardCopy" data-clipboard={@uri}>Copy URI</a>
```
Comment (jamilbk) [inline apps/fz_http/lib/fz_http_web/live/mfa_live/register_steps_component.ex]: Sorry for not clarifying — I think we probably want this to a be “Can’t scan the QR code? Click here to show the secret key” which displays a `<pre><code>` block containing the secret when clicked (that should make it easy for the user to either type it in manually or highlight and copy-paste).

I *think* this key should be the same data stored in the QR code. If you added it to Google Authenticator for example or another password manager it should accept it and start generating valid codes.

Comment (princemaple) [reply apps/fz_http/lib/fz_http_web/live/mfa_live/register_steps_component.ex]: 👌 Will get back to this tomorrow.

Comment (jamilbk) [reply apps/fz_http/lib/fz_http_web/live/mfa_live/register_steps_component.ex]: 👍🏼

Comment (princemaple) [reply apps/fz_http/lib/fz_http_web/live/mfa_live/register_steps_component.ex]: Please check the updated screenshots. I don't feel like I need to hide it (behind a click).

**Review by jamilbk — APPROVED** [review]
LGTM!

**Pull request merged**